doc/sphinx/source/changelog.rst

```diff
@@ -3,6 +3,112 @@
 Changelog
 =========
 
+.. _changelog-v2-9-0:
+
+v2.9.0
+------
+
+Highlights
+~~~~~~~~~~
+
+-  A new :ref:`diagnostic <api.esmvaltool.diag_scripts.seaborn_diag>` has been
+   added to provide a high-level interface to
+   `seaborn <https://seaborn.pydata.org/>`__,
+   a Python data visualization library based on
+   `matplotlib <https://matplotlib.org/>`__.
+   See the :ref:`recipe documentation <recipes_seaborn_diag>` for more
+   information.
+
+-  We have included a new recipe and diagnostic that represent the major
+   physical processes that describe Arctic-midlatitude teleconnections and
+   provide the basis for the CMIP6 model evaluation for the further application
+   of causal discovery.
+   The results are discussed in the article
+   `"Causal model evaluation of Arctic-midlatitude teleconnections in CMIP6" <https://essopenarchive.org/doi/full/10.1002/essoar.10512569.1>`__
+   by Galytska et al. (in review in Journal of Geophysical Research: Atmospheres).
+
+-  It is now possible to use the
+   `Dask distributed scheduler <https://docs.dask.org/en/latest/deploying.html>`__,
+   which can
+   `significantly reduce the run-time of recipes <https://github.com/ESMValGroup/ESMValCore/pull/2049#pullrequestreview-1446279391>`__.
+   Configuration examples and advice are available in the
+   :ref:`ESMValCore documentation <esmvalcore:config-dask>`.
+   If configured, the Dask distributed scheduler will also be used by diagnostic
+   scripts written in Python, so make sure to use
+   `lazy data <https://scitools-iris.readthedocs.io/en/latest/userguide/real_and_lazy_data.html#real-and-lazy-data>`__
+   where you can in your (new) diagnostics.
+   More work on improving the computational performance is planned, so please
+   share your experiences, good and bad, with this new feature in
+   `ESMValGroup/ESMValCore#1763 <https://github.com/ESMValGroup/ESMValCore/discussions/1763>`__.
+
```

Comment:
let's have us a geeky highlight too - Python 311 perhaps?

Reply:
I reckoned the dask stuff was already pretty geeky ;-)

Reply:
that's glamorous geeky, I meant something Gilbert-geeky :grinning: